clients/venice-admin-tool/src/test/java/com/linkedin/venice/TestDataRecoveryClient.java

```diff
@@ -66,6 +90,46 @@ private void verifyExecuteRecoveryResults(boolean isSuccess) {
     }
   }
 
+  private void estimateRecovery() {
```

Comment:
I guess we need to add `@Test` before `estimateRecovery` so that Testng would pick it up to run, don't we?